lib/hashie/mash.rb

```diff
@@ -149,6 +102,18 @@ def initialize(source_hash = nil, default = nil, &blk)
       default ? super(default) : super(&blk)
     end
 
+    # Creates a new anonymous subclass with key conflict
+    # warnings disabled. You may pass an array of method
+    # symbols to restrict the warnings blacklist to.
+    # Hashie::Mash.quiet.new(hash) all warnings disabled.
+    # Hashie::Mash.quiet(:zip).new(hash) only zip warning
+    # is disabled.
+    def self.quiet(*method_keys)
```

Comment:
Do we care to give someone the ability to name these quiet subclasses?

Reply:
I had considered it, especially since using it this way makes the error a little obtuse in the event they're only silencing a single error. It outputs an anonymous class inspect but still should have the stacktrace